src/config_wrapper.cpp

```diff
@@ -9,6 +9,15 @@ config_wrapper::config_wrapper(eosio::name self) : _self(self), _config(self, se
     if(_exists) {
         _cached_config = _config.get();
     }
+    if (!_cached_config.consensus_parameter.has_value()){
+        _cached_config.consensus_parameter = consensus_parameter_type();
+    }
+    std::visit([&](auto &v) {
+        if (v.minimum_gas_price == 0) {
+            v.minimum_gas_price = _cached_config.gas_price;
+            // don't set dirty, as trxs can be read-only
+        }
+    }, _cached_config.consensus_parameter->current);
```

Comment:
Change this to:

```cpp
if (!_cached_config.consensus_parameter.has_value()) {
   _cached_config.consensus_parameter = consensus_parameter_type();
   std::visit([&](auto &v) {
      v.minimum_gas_price = _cached_config.gas_price;
   }, _cached_config.consensus_parameter->current);
   // Don't set dirty because action can be read-only.
}
```